packages/protocol/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "@trezor/protocol",
```

Comment:
Name `protocol` seams too generic. What about binary-protocol?

Reply:
dunno, there is already something called "trezor protocol" here https://github.com/trezor/trezor-firmware/blob/ef2d2a3e7587fe90ebf56454a8486207dada89c5/common/protob/protocol.md?plain=1#L4